src/flicking.js

```diff
@@ -1318,6 +1321,68 @@ export default class Flicking extends Mixin(Component).with(EventHandler) {
 	}
 
 	/**
+	 * Get current flicking status
+	 * @ko 현재의 상태 값을 반환한다.
+	 * @method eg.Flicking#getStatus
+	 * @param {Boolean} stringify Set true if want get stringified status value <ko>상태 값을 문자열로 전달받고자 하는지 여부</ko>
+	 * @return {{panel: {index: (*|number), no: (*|number), currIndex: number, currNo: number}, $list}}
+	 */
+	getStatus(stringify) {
+		const panel = this._conf.panel;
+		const rxStyle = /(transform|left|top|will-change|box-sizing|width):[^;]*;/g;
+		const status = {
+			// current panel position
+			panel: {
+				index: panel.index,
+				no: panel.no,
+				currIndex: panel.currIndex,
+				currNo: panel.currNo
+			},
+
+			// panel's html
+			$list: panel.$list.map(v => ({
+				style: v.style.cssText.replace(rxStyle, "").trim(),
```

Comment:
- Do you have to check this out? `cssText` of panel isn't restore.
```js
v.style.cssText.replace(rxStyle, "").trim()
```

- Does `cssText` of base element is not save?